src/providers/mssql/qgsmssqlnewconnection.cpp

```diff
@@ -337,6 +339,66 @@ void QgsMssqlNewConnection::onCurrentDataBaseChange()
   mSchemaModel.setSchemasSetting( newSchemaSettings );
 }
 
+void QgsMssqlNewConnection::onExtentFromGeometryToggled( bool checked )
+{
+  if ( !checked )
+  {
+    bar->clearWidgets();
+    return;
+  }
+
+  if ( !testExtentInGeometryColumns() )
+    bar->pushWarning( tr( "Use extent in Geometry columns" ), tr( "Extent columns not found." ) );
```

Comment:
I would suggest to give more information e.g. like this:
```suggestion
    bar->pushWarning( tr( "Use extent from geometry_columns table" ), tr( "Extent columns (qgis_xmin, qgis_ymin, qgis_xmax, qgis_ymax) not found." ) );
```

- especially that we are talking about geometry_columns table, not just some geometry columns which can be confusing
- also show the names of the expected columns to the user